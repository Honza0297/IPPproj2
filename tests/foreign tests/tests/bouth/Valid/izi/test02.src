.IPPcode19
call foo
createframe
defvar TF@a
pushframe
move LF@a int@10
write LF@a
write string@\010
popframe
write TF@a
write string@\010
write string@řetězec\032s\032lomítkem\032\092\032a\010novým\035řádkem
write string@\010

exit int@0


label foo
createframe
defvar TF@a
pushframe
move LF@a int@5
write LF@a
write string@\010
popframe
write TF@a
write string@\010
return
