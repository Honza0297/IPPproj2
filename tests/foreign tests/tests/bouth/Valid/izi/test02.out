5
5
10
10
řetězec s lomítkem \ a
novým#řádkem
